management/mail_log.py

```diff
@@ -838,12 +837,12 @@ def print_header(msg):
     args = parser.parse_args()
 
     if args.startdate is not None:
-        START_DATE = args.startdate
+        END_DATE = args.startdate
         if args.timespan == 'today':
             args.timespan = 'day'
-        print("Setting start date to {}".format(START_DATE))
+        print("Setting start date to {}".format(END_DATE))
```

Comment:
Maybe we should rewrite this, otherwise this line seems - from a semantical perspective - misleading, i.e. the text says "start date", but we format it with the end-date. Otherwise I think it's cool that you switched END_DATE and START_DATE since it was a bit confusing to understand the sourcecode before :)